go/generate.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIR=../solidity
+
+# Source contracts for binding generation.
+# They should include the subdirectory prefix if any.
+FILES="DepositLog.sol deposit/Deposit.sol"
+
+SOLIDITY_DIR=$DIR SOLIDITY_FILES=$FILES make
```

Comment:
Can we avoid having this file and put it in `gen.go`? I believe this is the strategy we use for all the other cases so far.

Reply:
I think having it as a separate file will be more readable once we will have more items in the `FILES` variable.

Reply:
Agreed though until there are just two files, I'd keep it in `gen.go` just for consistency. Maybe we'll find some way in the future keep there more than two files in a clean way.

Reply:
Ok, I'll change!